constants/ChangedChatErrors.json

```diff
@@ -103,6 +103,12 @@
         "TPS calculation got an error"
       ],
       "fixed_in": "1.12.0"
+    },
+    {
+      "message_starts_with": [
+        "Could not find gemstone slot price for"
+      ],
+      "fixed_in: "2.3.0"
```

Comment:
json wronh

Reply:
I blame whatever the hell my intellij is smoking rn